### PR TITLE
Simplified op declarations

### PR DESCRIPTION
**Pull request by radarhere**

### CODE REVIEW

File: src/PIL/TiffImagePlugin.py [hunk 1]
```diff
@@ -370,42 +370,37 @@ def delegate(self, *args):
 
         return delegate
 
-    """ a = ['add','radd', 'sub', 'rsub','div', 'rdiv', 'mul', 'rmul',
-             'truediv', 'rtruediv', 'floordiv',
-             'rfloordiv','mod','rmod', 'pow','rpow', 'pos', 'neg',
-             'abs', 'trunc', 'lt', 'gt', 'le', 'ge', 'nonzero',
-             'ceil', 'floor', 'round']
-        print("\n".join("__%s__ = _delegate('__%s__')" % (s,s) for s in a))
-        """
```
Comment (hugovk) [inline src/PIL/TiffImagePlugin.py]: Did this do nothing?

File: src/PIL/TiffImagePlugin.py [hunk 2]
```diff
@@ -370,42 +370,37 @@ def delegate(self, *args):
...
+        "lt",
+        "gt",
+        "le",
+        "ge",
+        "nonzero",
+        "ceil",
+        "floor",
+        "round",
+    ):
+        vars()["__" + op + "__"] = _delegate("__" + op + "__")
```
Comment (hugovk) [inline src/PIL/TiffImagePlugin.py]: Hmm, this makes it a bit easier to add new ops, but I think it's less readable.

A comment to make it clear what's happening may help, eg.

```python
# Equivalent to __add__ = _delegate("__add__") etc.
```

File: src/PIL/TiffImagePlugin.py (hunk 1 above)
Comment (radarhere) [reply src/PIL/TiffImagePlugin.py]: I think it's intended as instructions for how to generate the declarations

File: src/PIL/TiffImagePlugin.py (hunk 2 above)
Comment (radarhere) [reply src/PIL/TiffImagePlugin.py]: Yes, I could see arguments either way for this. I've added a commit with your comment.

Comment (hugovk) [reply src/PIL/TiffImagePlugin.py]: Thanks for the update. I'm +0 in this, so I'll leave it up to you (or someone else) :)

**Comment by radarhere** [comment]
Closing due to debate over whether this is simpler or not.

**Pull request closed without merging**